y2020/ex16/README.md

```diff
@@ -1,64 +1,65 @@
---- Day 16: Ticket Translation ---
+# Day 16: Ticket Translation
 As you're walking to yet another connecting flight, you realize that one of the legs of your re-routed trip coming up is on a high-speed train. However, the train ticket you were given is in a language you don't understand. You should probably figure out what it says before you get to the train station after the next flight.
 
 Unfortunately, you can't actually read the words on the ticket. You can, however, read the numbers, and so you figure out the fields these tickets must have and the valid ranges for values in those fields.
 
 You collect the rules for ticket fields, the numbers on your ticket, and the numbers on other nearby tickets for the same train service (via the airport security cameras) together into a single document you can reference (your puzzle input).
 
-The rules for ticket fields specify a list of fields that exist somewhere on the ticket and the valid ranges of values for each field. For example, a rule like class: 1-3 or 5-7 means that one of the fields in every ticket is named class and can be any value in the ranges 1-3 or 5-7 (inclusive, such that 3 and 5 are both valid in this field, but 4 is not).
+The rules for ticket fields specify a list of fields that exist somewhere on the ticket and the valid ranges of values for each field. For example, a rule like class: `1-3` or `5-7` means that one of the fields in every ticket is named class and can be any value in the ranges `1-3` or `5-7` (inclusive, such that `3` and `5` are both valid in this field, but `4` is not).
 
 Each ticket is represented by a single line of comma-separated values. The values are the numbers on the ticket in the order they appear; every ticket has the same format. For example, consider this ticket:
 
-.--------------------------------------------------------.
```

Comment:
I would have kept the dashes lines

Reply:
Not sure if it's worth making this an actual table .. the inputs don't look like this, so this representation is more for people without imagination I guess?

Reply:
no,no, I mean inside the triple backquote sections

Reply:
Yep. It had that line to make it look like an actual paper ticket or something. But I'm thinking whether it's worth keeping it like that, or just making it look like the inputs. But it's prolly best to leave it as it is (even if it doesn't provide value to the problem)